keps/sig-network/2086-service-internal-traffic-policy/kep.yaml

```diff
@@ -34,7 +34,7 @@ milestone:
 # The following PRR answers are required at alpha release
 # List the feature gate name and the components for which it must be enabled
 feature-gates:
-  - name: ServiceInternalTrafficPolicy
+  - name: ServiceITrafficPolicy
     components:
       - kube-apiserver
```

Comment:
Nit: This will also need to be read from kube-controller-manager now that topology is dependent on this feature gate.